mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -854,6 +856,8 @@ pub struct NegotiationDataV2 {
     started_at: u64,
     payment_locktime: u64,
     secret_hash: Vec<u8>,
+    #[serde(serialize_with = "H264::serialize_as_byte_seq")]
+    #[serde(deserialize_with = "H264::deserialize_as_bytes")]
```

Comment:
decided to use new ser/der functions instead of modifying the old ones as the old ones needed for backward compatibility for swap files ref https://github.com/KomodoPlatform/komodo-defi-framework/blob/39515a9f3ea1089bb462e99c8cafb1049a920dbd/mm2src/mm2_main/src/lp_swap/taker_swap.rs#L660